test-suite/misc/coqtop_print-mod-uid.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+export COQBIN=$BIN
+export PATH=$COQBIN:$PATH
+
+TMP=`mktemp -d`
+cd $TMP
+
+function assert_eq() {
+  if [ "$1" != "$2" ]; then
+    echo "coqtop -print-mod-uid" $1 "!=" $2
```

Comment:
The error message will be quite strange, since it will show the result of `coqtop` as an argument of `coqtop`.